### DISCUSSION TITLE
Provide a way to store additional package meta information; Sync data with awesome-cpp

### DISCUSSION THREAD

**Discussion by Fleischner** [microsoft/vcpkg#17793] [Ideas]
https://github.com/fffaraz/awesome-cpp/issues/789
#6154

- [ ] vcpkg: choose which information belongs into the port file
- [ ] vcpkg: choose a way to store additional meta information (eg: build status, operating system support, ...)

**Comment by myd7349** [comment]
How about adding a `Topic`/`Category`/`Keywords` field? For example, all logging libraries in `vcpkg` can be tagged with `Logging`.
